### DISCUSSION TITLE
Welcome to Notepads Discussions!

### DISCUSSION THREAD

**Discussion by 0x7c13** [0x7c13/Notepads#813] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soumyamahunt** [comment]
@JasonStein the pinned issues aren't actually issues, shouldn't they belong here??

**Reply by 0x7c13** [reply]
Yeah, I will move them here.

**Comment by ocrey** [comment]
@JasonStein Congratulations the project is amazing , i love use it   but exists the possibility to add some extension like vim keybinds ?

**Reply by 0x7c13** [reply]
No plan for adding vim like editing support.
